### DISCUSSION TITLE
Review of the mapping of ExPaNDS and PaNOSC projects activities

### DISCUSSION THREAD

**Discussion by servansod** [ExPaNDS-eu/ExPaNDS#15] [Show and tell]
Dear All,

Last year during our joint annual meeting, with Jordi, we did the exercise to map our two projects activities.
This preliminary work was then detailed and put into **[this wiki page](https://github.com/ExPaNDS-eu/ExPaNDS/wiki/Mapping-ExPaNDS-and-PaNOSC-activities)**.

I never took the time to ask for a proper review by all of you. 
Now the Commission is asking us to include this information in our ExPaNDS mid-term review report so this time I will please ask you to **check, contribute or comment** wherever needed. 

If you are a member of ExPaNDS organisation already, you can make changes directly in the wiki. Otherwise you can a) ask me to be added b) reply to this discussion with your suggestions.

Feedback by :date: 21/07/2021 would be great. 

Many thanks in advance for your help!

**Comment by RKrahl** [comment]
One comment: on the wiki page, Parallel work / WP3s / Metadata catalogues lists "number of facility Data Catalogues with the OAI-PMH protocol to allow harvesting into OpenAIRE" as KPI. Not sure whether this is appropriate. In WP3 we have a number of KPIs to measure the progress of the facilities, being harvested with OAI-PMH is only one of them. In case of doubt it might be good to confirm this with PSI, the task leader of 3.1, because these KPIs were the outcome of the landscape analysis been done in Task 3.1.

**Reply by servansod** [reply]
True, and I will check with Alun. The thing is I was looking in the list of **common** KPIs between the 2 projects, and I don't think the overall evaluation of the readiness of the facilities that was made in the frame of task 3.1 is something PaNOSC is also doing, at least not as a KPI.
Thanks for checking and for your edits Rolf, always appreciated.

**Comment by wiggle120** [comment]
Three questions: 
1. Under 'Common work' WP8/5 Training - the KPI here refers to number of staff participating in training workshops.  Who counts as staff? Does this mean ExpaNDS/PaNOSC 'staff'?  What about other people who attend our workshops? 
2. Under 'Parallel work' WP2 FAIR data policy - we in WP2 have always been clear that while we can consult with facilities around data policy (and indeed, we have done so now, extensively), we cannot force facilities to adopt the data policy framework recommendations.  Indeed, it is not in our DoW to do the latter. Could we remove 'to adopt' from this sentence, and change it to something else like 'to consider'?
3. Under 'Extras'  WP2 FAIR assessment - my feeling is that the statements here are possibly a bit ambitious?  Of course, we have yet to start this task, so it is somewhat difficult to know for sure, but it does seem that this task in WP2 is more focussed on surveying what is out there as possibilities for FAIR assessment and beginning initial self-assessment exercises at national PaN RIs?

**Reply by servansod** [reply]
Thanks a lot Abigail for your review.
1. I think it's meant as a generic 'staff' including everyone but this would be a question to send back to WP5/WP8
2. fair enough, I will change from "to adopt" to "to consider" to be more inclusive/accurate
3. ok then how about changing this statement from:
> - ~~Define the FAIR assessment process applicable to our NRIs~~
> - ~~Act as a use case for community tools like F-UJI and/or CoreTrustSeal~~

to:

> - Survey available FAIR assessment tools (e.g FAIRsFAIR's F-UJI) and assess their applicability to PaN
> - Lead an open self-assessment exercise of the NRIs against these schemes

?

**Reply by wiggle120** [reply]
Thanks, Sophie - re point 3, what you suggest looks reasonable to me, but because Brian understands task 2.5 better than me, it would be helpful to have him take a look as well.  As far as I know, though, I think the rewording you provide is pretty much what we hope to do! :)

**Reply by servansod** [reply]
Changes are now done in the wiki accordingly :heavy_check_mark:, so we have a clean version for Brian to check. I'll bring it up this afternoon.

**Comment by UweKonrad** [comment]
Dear Abigail,

 

regarding Question 1: I think staff means facility staff, not project staff. This includes other people (e.g. facility users?).

Q2: If there is no plan to do significant policy workshops in the second half of the project, than this is the WP2 decision. 

Policy Workshops probably might be needed at the individual facilities, that should at least be a recommendation?

Q3: I agree.

 

Thanks and best regards.

Uwe

 

 

Von: Abigail McBirnie ***@***.***> 
Gesendet: Montag, 19. Juli 2021 14:04
An: ExPaNDS-eu/ExPaNDS ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: Re: [ExPaNDS-eu/ExPaNDS] Review of the mapping of ExPaNDS and PaNOSC projects activities (#15)

 

Three questions:

1.	Under 'Common work' WP8/5 Training - the KPI here refers to number of staff participating in training workshops. Who counts as staff? Does this mean ExpaNDS/PaNOSC 'staff'? What about other people who attend our workshops?
2.	Under 'Parallel work' WP2 FAIR data policy - we in WP2 have always been clear that while we can consult with facilities around data policy (and indeed, we have done so now, extensively), we cannot force facilities to adopt the data policy framework recommendations. Indeed, it is not in our DoW to do the latter. Could we remove 'to adopt' from this sentence, and change it to something else like 'to consider'?
3.	Under 'Extras' WP2 FAIR assessment - my feeling is that the statements here are possibly a bit ambitious? Of course, we have yet to start this task, so it is somewhat difficult to know for sure, but it does seem that this task in WP2 is more focussed on surveying what is out there as possibilities for FAIR assessment and beginning initial self-assessment exercises at national PaN RIs?

**Reply by thomasholmrod** [reply]
Regarding #1: Staff refers to staff at facilities excl users. Depending on the workshops, we can discuss if it would make more sense to just say participants.

**Comment by pozsa** [comment]
Hello @servansod 

WP4s | PaN portal | Develop a portal for single entry point to all PaN data catalogues and services | Deploy the portal at national facilities. KPI: number of facilities deploying the PaN portal
-- | -- | -- | --

The above is in complementary work.

I think we will do poorly on this KPI given the recent developments from panosc. Is the KPI set in stone?
What do you think about the following wording for expands focus: "Expose data analysis services via PaN portal or directly to EOSC" ?

**Reply by UweKonrad** [reply]
Yes, this is a very helpful proposal!

**Reply by servansod** [reply]
Thanks for your review @pozsa !
Ok I see your point, let me think about how to take this into account. Because the fact that we deploy data services to EOSC is not really complementary from PaNOSC's work and would better fit into the parallel work section below:

WP4s | Data analysis pipelines | Make the data analysis FAIRer and externally reachable
-- | -- | --

How about the following

Under Complementary work:
> ~~Deploy the portal at national facilities. KPI: number of facilities deploying the PaN portal~~
> Deploy the PaN portal at national facilities wherever possible (OpenStack infrastructure in place)

Under Parallel work:
> ~~Make the data analysis FAIRer and externally reachable~~
> - Make the data analysis pipelines FAIRer and externally reachable
> - Expose them to the EOSC via the PaN portal or directly

?

**Reply by pozsa** [reply]
parallel work change sounds good to me.

complementary change, I think it's better, but I still wonder how many RIs are actually planning to deploy VISA among the ones that run OpenStack.

**Reply by servansod** [reply]
Okay @pozsa I understand. But I think it's not that bad if the target for this KPI is only 2 or 3 facilities out of 10. The KPIs are not set in stone no, but the advantage of this one is that it is common with PaNOSC. So if their target is 6/6 ESFRI deploy VISA, and it can be 2 or 3 more thanks to our NRIs, well it's already something.
I'll leave it in the wiki for now, it's not going in the report to the EU anyway. If you can discuss it within WP4 when Anton is back, setting this target, that'd be great.

**Comment by pozsa** [comment]
WP3s | Common search API
-- | --

Wouldn't this fall under complementary? (currently its in common)
panosc focus: Define common search api and develop consumer layer for the apis.
expands focus: Implement scicat and icat search apis based on defined endpoints.

?

**Reply by servansod** [reply]
Good point, I thought ExPaNDS also participated in the definition of the search API but even if we did, it was maybe not to the level where we can say it's a common task. I'll amend with your suggestion, maybe opening to custom data catalogues too, in addition to SciCat/ICAT.

**Reply by pozsa** [reply]
Hi @servansod, sorry about the changes back-n-forth.
I went through with Alun on this point and he told me that expands did collaborate with panosc on defining the common API. Could you please move it back to common work?

the below part in parallel work covers the separate scicat/icat implementations.

WP3s | Metadata catalogues | Evolve to community data catalogues (e.g. ICAT and SciCat) and make them externally reachableKPI: number of facility Data Catalogues with the OAI-PMH protocol to allow harvesting into OpenAIRE
-- | -- | --

sorry again

**Reply by servansod** [reply]
No probs, I reverted changes. Let's see what Tobias has to say too.

**Comment by martaguti** [comment]
Hi @servansod 
Please see the comments and suggestions. If agreed I would be happy to update the tables myself. 

The entry "Cloud resources" on the Extra table also applies to ExPaNDS , e.g. we have deployed the ontologies on the EGI infrastructure for ExPaNDS and Jupiter Notebooks for PaNOSC.

1. Proposal to remove Cloud resources from Extra table and insert instead in the Common Work

WPs|Domain|Common activities|Links
-|-|-|-
WP6/WP3|**Cloud resources**|Offer cloud resources through the EGI FedCloud for several PaN use cases|- [ExPaNDS Ontology](https://ontoportal.egi.eu/)<br>- [PaNOSC Jupyter notebooks for data analytics](https://notebooks-panosc.fedcloud-tf.fedcloud.eu/hub/login)

2. Is AAI integration in the Extras table not meant to be a common activity for both projects ? WP6 of PaNOSC and WP3/4/5 for ExPaNDS.  Should that entry be in the common work. In any case we would like to add to the description __integrate UmbrellaID with EGI Check-in__

3. Propose to add the following entry to the commons table

WPs|Domain|Common activities|Links
-|-|-|-
WP8/WP5|**On demand e-learning platform**|Provision of adhoc e-learning platform on the EGI infrastructure with additional capacity during training peaks.|

**Reply by servansod** [reply]
Thanks @martaguti :)
- 1: agreed - please do this change.
- 2: well on paper, and much to Jean-François's dismay, there is nothing in ExPaNDS DoA about AAI - so if we do integrate UmbrellaID in the log-in options of ExPaNDS-related services, it would be on a voluntary / in-kind basis. So I would rather keep it in "Extras". Ok to add the addition on EGI check-in.
- 3: great - please do this change too.

**Reply by martaguti** [reply]
thanks for your feedback @servansod tables updated as agreed.

**Comment by thomasholmrod** [comment]
Regarding the training work packages. There are also development of the [training catalogue](https://github.com/orgs/pan-training/projects/3) (ExPaNDS) and [e-learning platform](https://github.com/orgs/pan-training/projects/2) (PaNOSC) and the [_joint_ web-site](https://github.com/pan-training/WP8-Documents/issues/9) hosting both these solutions.

**Reply by thomasholmrod** [reply]
Wrt to the wiki-page:

Under common work I would put: Joint pan-training web-site hosting training catalogue and e-learning platform

Under complementary work: ExPaNDS: training catalogue. PaNOSC: e-learning platform.

Regarding workshops, we have not really organized or developed joint workshops. They have either been organized by PaNOSC or ExPaNDS although both sides have been in the loop due to our regular joint meetings. So I gues this can be listed as common or complementary work.

**Reply by servansod** [reply]
Thanks @thomasholmrod for checking and for mentioning the common training "landing page". I made the edits :+1: . For the workshops, I left it in "common" but removed "jointly", which seemed like a good compromise. And I simplified the "complementary work" line as you suggested.
